internal/controller/sync.go

```diff
@@ -107,7 +107,7 @@ func (r *Route) sync(ctx context.Context, req reconcile.Request, route *routev1.
 		if err != nil {
 			return result, err
 		}
-		r.eventRecorder.Event(route, corev1.EventTypeNormal, ReasonIssuing, "Created new CertificateRequest for Route %s")
+		r.eventRecorder.Event(route, corev1.EventTypeNormal, ReasonIssuing, "Created new CertificateRequest")
```

Comment:
This is the one that you mean would be on the route anyway? So no need to include the name of the route... on the route already?

Reply:
> This is the one that you mean would be on the route anyway? So no need to include the name of the route... on the route already?

Yea, exactly that, the event is raised against the route, so makes no/little sense of including the name of the route in the event